src/sentry/scim/endpoints/members.py

```diff
@@ -274,10 +273,18 @@ def patch(self, request: Request, organization, member):
             # we only support setting active to False which deletes the orgmember
             if self._should_delete_member(operation):
                 self._delete_member(request, organization, member)
+                metrics.incr(
+                    "sentry.scim.member.update",
+                    sample_rate=1.0,
+                    tags={"organization": organization},
+                )
                 return Response(status=204)
             else:
                 raise SCIMApiError(detail=SCIM_400_INVALID_PATCH)
 
+        metrics.incr(
+            "sentry.scim.member.update", sample_rate=1.0, tags={"organization": organization}
```

Comment:
nit: you can create tags variable and use it everywhere.

Reply:
The organization variable is an argument to each endpoint so I don't think there a way to have set a global tags variable